pkg/validator/register/register.go

```diff
@@ -0,0 +1,15 @@
+package register
+
+import (
+	_ "github.com/mt-sre/addon-metadata-operator/pkg/validator/am0001"
+	_ "github.com/mt-sre/addon-metadata-operator/pkg/validator/am0002"
+	_ "github.com/mt-sre/addon-metadata-operator/pkg/validator/am0004"
+	_ "github.com/mt-sre/addon-metadata-operator/pkg/validator/am0005"
+	_ "github.com/mt-sre/addon-metadata-operator/pkg/validator/am0006"
+	_ "github.com/mt-sre/addon-metadata-operator/pkg/validator/am0007"
+	_ "github.com/mt-sre/addon-metadata-operator/pkg/validator/am0008"
+	_ "github.com/mt-sre/addon-metadata-operator/pkg/validator/am0009"
+	_ "github.com/mt-sre/addon-metadata-operator/pkg/validator/am0010"
+	_ "github.com/mt-sre/addon-metadata-operator/pkg/validator/am0011"
+	_ "github.com/mt-sre/addon-metadata-operator/pkg/validator/am0012"
+)
```

Comment:
Should we document this new init process. If I understand correctly, the idea is to dummy import this register package, which dummy imports all validators, so that all init funcs are ran? Probably need to review the whole `validators_test.md` doc.